### PR TITLE
Implement autofix for B013

### PR DESCRIPTION
**Pull request by harupy**
#389

### CODE REVIEW

File: src/flake8_bugbear/plugins/redundant_tuple_in_exception_handler.rs [hunk 1]
```diff
@@ -1,20 +1,35 @@
...
+                    let mut check = Check::new(
+                        CheckKind::RedundantTupleInExceptionHandler(elt.to_string()),
                         Range::from_located(type_),
-                    ));
+                    );
+                    if checker.patch(check.kind.code()) {
+                        let mut generator = SourceGenerator::new();
+                        if let Ok(()) = generator.unparse_expr(elt, 0) {
+                            if let Ok(content) = generator.generate() {
+                                check.amend(Fix::replacement(
```
Comment (harupy) [inline src/flake8_bugbear/plugins/redundant_tuple_in_exception_handler.rs:24]:
```python
try:
    pass
except (ValueError,):
    pass
```

is fixed to:

```python
try:
    pass
except (ValueError):
    pass
```

Is it possible to remove unnecessary parens?

Comment (JonathanPlasse) [reply src/flake8_bugbear/plugins/redundant_tuple_in_exception_handler.rs:24]: Yes

Comment (charliermarsh) [reply src/flake8_bugbear/plugins/redundant_tuple_in_exception_handler.rs:24]: I think it will require some improvements to the source code generator.

Comment (charliermarsh) [reply src/flake8_bugbear/plugins/redundant_tuple_in_exception_handler.rs:24]: I can take a look at it...

Comment (charliermarsh) [reply src/flake8_bugbear/plugins/redundant_tuple_in_exception_handler.rs:24]: Oh, it looks like the issue here is actually in the RustPython locations, which exclude the parens.

File: src/flake8_bugbear/plugins/redundant_tuple_in_exception_handler.rs [hunk 2]
```diff
@@ -1,20 +1,85 @@
...
+        Ok(Range {
+            location,
+            end_location,
+        })
+    } else {
+        Err(anyhow::anyhow!(
+            "Unable to find left and right parentheses."
+        ))
+    }
+}
```
Comment (harupy) [inline src/flake8_bugbear/plugins/redundant_tuple_in_exception_handler.rs:52]: @charliermarsh Nice! Can we use this approach to find the range of a class name?

```python
class MyClass(...):
      ^^^^^^^
      This
```

Comment (charliermarsh) [reply src/flake8_bugbear/plugins/redundant_tuple_in_exception_handler.rs:52]: Yeah probably!

I'm also wondering if `Ident` fields like the `ClassDef` name should have locations added to them in the parser...

**Pull request merged**